app/src/main/java/fr/free/nrw/commons/upload/UploadActivity.java

```diff
@@ -309,19 +309,29 @@ public void setRightCardVisibility(boolean visible) {
     }
 
     @Override
-    public void setBottomCardVisibility(@UploadPage int page) {
+    public void setBottomCardVisibility(@UploadPage int page, int uploadCount) {
         if (page == TITLE_CARD) {
             viewFlipper.setDisplayedChild(0);
         } else if (page == CATEGORIES) {
             viewFlipper.setDisplayedChild(1);
+            TextView categoriesSubtitle = viewFlipper.findViewById(R.id.categories_subtitle);
+            setSubtitleVisibility(categoriesSubtitle, uploadCount);
```

Comment:
Is there need to create TextView each time setBottomCardVisibility is called. I think you can use BindView once.

Reply:
This TextView is in a included layout. I am not very familiar with ButterKnife. Is there a easy way to bind to a included layout's view?